plutarch-trace/impl/Plutarch/Trace/Disable.hs

```diff
@@ -0,0 +1,24 @@
+module Plutarch.Trace.Disable (ptrace', ptrace, ptraceIfTrue, ptraceIfFalse) where
+
+import Plutarch.Bool (PBool)
+import Plutarch.Prelude
+import Plutarch.String (PString)
+
+pf :: Term s (b :--> a :--> a)
+pf = phoistAcyclic $ plam $ \_ y -> y
+
+-- | A strict version of 'ptrace'.
```

Comment:
I don't think this documentation is needed.

Reply:
Do you think the traces should be removed from the implementation and only kept in the signature? I actually have no idea how hoogle works with signatures (that is, if we ever decide to generate hoogle docs for this).

Reply:
The documentation should probably only be on the signatures.

Reply:
Done!